examples/disrupt-http.js

```diff
@@ -0,0 +1,81 @@
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+import { PodDisruptor } from '../src/pod.js';
+import { DeploymentHelper } from '../src/helpers.js';
+import  http from 'k6/http';
+
+const app = "httpbin"
+const image = "kennethreitz/httpbin"
+
+export function setup() {
+  const k8sClient = new Kubernetes()
+
+  // create a random namespace
+  const namespace = "k6-"+Math.random().toString(32).slice(2, 7);
+  k8sClient.namespaces.create({name: namespace})
+
+  // create a test deployment
+  const helper = new DeploymentHelper(k8sClient, app, namespace, image, 1)
+  helper.deploy()
+  
+  const ip = helper.expose()
+
+  // pass service ip to scenarios
+  return {
+    srv_ip: ip,
+    namespace: namespace,
+  }
+}
+
+export function teardown(data) {
+  const k8sClient = new Kubernetes()
+  k8sClient.namespaces.delete(data.namespace)
+}
+
+export function disrupt(data) {
+  const k8sClient = new Kubernetes()
+
+  // delay traffic from one random replica of the deployment
+  const podDisruptor = new PodDisruptor(
+    k8sClient,
+    {
+      namespace: data.namespace,
+      selector: { app: app},
+      picker: 'random'
+    }
+  )
```

Comment:
new PodDisruptor().disruptHttp feels wrong 👀 

At least my mind crashes a bit with having disrupt in the method too, but yeah, big nit.

Reply:
What will you suggest? The method does potentially several things: delay requests, inject errors.

Reply:
We instantiate a PodDisruptor, with some selectors, which are our targets. 

Then, maybe, it makes sense to pick the layer where the disruption happens. 

Example: `new PodDisruptor(<how we select>).http(<what we want to do to the layer>)`

Reply:
But yeah, proposal from the top of my mind, maybe it doesn't make sense 😄

Reply:
Interesting. I was just thinking about some similar idea, but in the constructuror, like:
```
new PodDisruptor(
 client,
 kind,
 selector
 )
 ```
 where kind is `http`, `network`, `resources`, etc.

Reply:
I like the idea of picking the kind, not sure where it's the best place, but yeah!

Agree. Let's not focus too much on it right now 👍